product_import/wizard/product_import.py

```diff
@@ -91,10 +91,22 @@ def parse_product_catalogue(self, product_file, product_filename):
         catalogue["attachments"][product_filename] = b64encode(product_file)
         return catalogue
 
+    @api.model
+    def _get_company(self, catalogue):
+        part = self._bdimport._match_partner(
+            catalogue["company"],
+            catalogue["chatter_msg"],
+            partner_type="contact",
+            domain=[("is_company", "=", True)],
+        )
+        return part.ref_company_ids[:1]
```

Comment:
This field comes from `account` module. I don't think we want this dependency.
You can search for companies instead.
Am I missing anything?